dolphinscheduler-ui/src/locales/zh_CN/resource.ts

```diff
@@ -54,31 +56,8 @@ export default {
   },
   udf: {
     udf_resources: 'UDF资源',
-    create_folder: '创建文件夹',
     upload_udf_resources: '上传UDF资源',
-    udf_source_name: 'UDF资源名称',
-    user_name: '所属用户',
-    whether_directory: '是否文件夹',
-    file_name: '文件名称',
-    file_size: '文件大小',
-    description: '描述',
-    create_time: '创建时间',
-    update_time: '更新时间',
-    operation: '操作',
-    yes: '是',
-    no: '否',
-    edit: '编辑',
-    download: '下载',
-    delete: '删除',
-    success: '成功',
-    folder_name: '文件夹名称',
-    upload: '上传',
-    upload_files: '上传文件',
-    file_upload: '文件上传',
-    delete_confirm: '确定删除吗?',
-    enter_keyword_tips: '请输入关键词',
-    enter_name_tips: '请输入名称',
-    enter_description_tips: '请输入描述'
```

Comment:
The keys should also be removed in the en_US file too.

Reply:
resolved